snapcraft/models/project.py

```diff
@@ -1265,16 +1287,16 @@ class Project(models.Project):
     )
     """The identifying name of the snap.
 
-    It must start with an ASCII character and can only contain lower case
-    letters, numbers, and hyphens. It must contain at least one letter and it
-    can't start or end with a hyphen. The maximum length is 40 characters.
+    It must start with an ASCII character and can only contain lower case letters,
+    numbers, and hyphens. It must contain at least one letter and it can't start or end
+    with a hyphen. The maximum length is 40 characters.
 
     The name must be unique if you want to `publish it to the Snap Store
-    <https://snapcraft.io/docs/releasing-your-app>`_.
+    <https://documentation.ubuntu.com/snapcraft/latest/how-to/publishing/publish-a-snap>`_.
```

Comment:
Why do you have `latest` in some urls and `stable` in others?

Reply:
Good catch. I believe it's because the docstring updates predate the RTD branch name switch. Updated all the fully-qualified internal references. 5cac99209